eoscostarica/lifebank
# fix(backend): remove yarn.lock files

Resolves #625 

### What does this PR do?
Remove yarn.lock file from webapp and hapi

### How should this be tested?
- Check directory /webapp and /hapi
- Check what yarn.lock file doesn't exist
